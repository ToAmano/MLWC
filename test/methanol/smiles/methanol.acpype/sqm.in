Run semi-empirical minimization
 &qmmm
  qm_theory='AM1', grms_tol=0.05, scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   6     C        0.9440        0.0230       -0.0920 
   8     O        0.4690        0.2700       -1.4030 
   1     H        0.5770       -0.9510        0.2400 
   1    H1        0.5740        0.7980        0.5830 
   1    H2        2.0370        0.0170       -0.0940 
   1    H3        0.8100        1.1410       -1.6710 

