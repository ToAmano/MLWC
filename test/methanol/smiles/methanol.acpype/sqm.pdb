ATOM      1  C   UNL     1       0.952   0.037  -0.093  1.00  0.00           C
ATOM      2  O   UNL     1       0.464   0.262  -1.397  1.00  0.00           O
ATOM      3  H   UNL     1       0.551  -0.966   0.200  1.00  0.00           H
ATOM      4  H1  UNL     1       0.578   0.820   0.614  1.00  0.00           H
ATOM      5  H2  UNL     1       2.070   0.022  -0.081  1.00  0.00           H
ATOM      6  H3  UNL     1       0.796   1.122  -1.680  1.00  0.00           H
