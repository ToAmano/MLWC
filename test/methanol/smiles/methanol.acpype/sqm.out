            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =  7

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       12
| QMMM:    diag iterations used for timing =       20
| QMMM:
| QMMM:              Internal diag routine = 0.003729 seconds
| QMMM:                 Dspev diag routine = 0.013063 seconds
| QMMM:                Dspevd diag routine = 0.009349 seconds
| QMMM:                Dspevx diag routine = 0.015027 seconds
| QMMM:                 Dsyev diag routine = 0.010986 seconds
| QMMM:                Dsyevd diag routine = 0.009520 seconds
| QMMM:                Dsyevr diag routine = 0.010165 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.001405 seconds
| QMMM:
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        0.9440    0.0230   -0.0920
  QMMM:     2        2      O        0.4690    0.2700   -1.4030
  QMMM:     3        3      H        0.5770   -0.9510    0.2400
  QMMM:     4        4      H        0.5740    0.7980    0.5830
  QMMM:     5        5      H        2.0370    0.0170   -0.0940
  QMMM:     6        6      H        0.8100    1.1410   -1.6710

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -37.8412  -26.9819  -18.2796  -15.4485  -15.2892  -12.4943  -11.1349    3.7777
   4.0894    4.2301    4.5224    6.2439

 Heat of formation   =        -57.06664046 kcal/mol  (       -2.47459522 eV)

 Total SCF energy    =     -11622.87970880 kcal/mol  (     -504.00588478 eV)
 Electronic energy   =     -24675.11011721 kcal/mol  (    -1069.99306696 eV)
 Core-core repulsion =      13052.23040841 kcal/mol  (      565.98718219 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.073
     2      O                 -0.326
     3      H                  0.098
     4      H                  0.053
     5      H                  0.053
     6      H                  0.195
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE     0.938    0.912    0.957    1.621
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        0.9515    0.0375   -0.0927
  QMMM:     2        2      O        0.4644    0.2622   -1.3971
  QMMM:     3        3      H        0.5513   -0.9657    0.1999
  QMMM:     4        4      H        0.5780    0.8200    0.6141
  QMMM:     5        5      H        2.0700    0.0219   -0.0809
  QMMM:     6        6      H        0.7958    1.1222   -1.6803

           --------- Calculation Completed ----------

